lib/src/modules/conditional.dart

```diff
@@ -267,14 +267,18 @@ class Sequential extends _Always {
   /// The input clocks used in this block.
   final List<Logic> _clks = [];
 
+  /// This sets the Sequential conditions to reset
+  Logic? reset;
+  Map<Logic, Logic> resetVal = {};
+
   /// Constructs a [Sequential] single-triggered by [clk].
   Sequential(Logic clk, List<Conditional> conditionals,
-      {String name = 'sequential'})
-      : this.multi([clk], conditionals, name: name);
+      {String name = 'sequential', Logic? reset})
+      : this.multi([clk], conditionals, name: name, reset: reset);
 
   /// Constructs a [Sequential] multi-triggered by any of [clks].
   Sequential.multi(List<Logic> clks, List<Conditional> conditionals,
-      {String name = 'sequential'})
+      {String name = 'sequential', this.reset})
```

Comment:
Maybe put the `reset` parameter before `name`? This does not affect the functionality (named parameters), but will keep it consistent with other modules that have `name` at the end.